packages/reporters/dev-server/src/Server.js

```diff
@@ -335,45 +335,35 @@ export default class Server extends EventEmitter {
     await this.applyProxyTable(app);
     app.use(finalHandler);
 
-    if (!this.options.https) {
-      this.server = http.createServer(app);
-    } else if (typeof this.options.https === 'boolean') {
-      this.server = https.createServer(
-        await generateCertificate(this.options.outputFS, this.options.cacheDir),
-        app
-      );
-    } else {
-      this.server = https.createServer(
-        await getCertificate(this.options.inputFS, this.options.https),
-        app
-      );
-    }
-
-    this.server.listen(this.options.port, this.options.host);
+    let {server, stop} = await createHTTPServer({
+      cacheDir: this.options.cacheDir,
+      https: this.options.https,
+      inputFS: this.options.inputFS,
+      listener: app,
+      outputFS: this.options.outputFS
+    });
+    this.server = server;
```

Comment:
`this.server` isn't actually used anymore (aside from clearing it below). Should we just remove it?